modules/core/src/types.cpp

```diff
@@ -173,13 +173,13 @@ RotatedRect::RotatedRect(const Point2f& _point1, const Point2f& _point2, const P
 void RotatedRect::points(Point2f pt[]) const
 {
     double _angle = angle*CV_PI/180.;
-    float b = (float)cos(_angle)*0.5f;
-    float a = (float)sin(_angle)*0.5f;
+    double b = cos(_angle)*0.5;
+    double a = sin(_angle)*0.5;
```

Comment:
do we still need changes here?

Reply:
Actually, we don't need to change it, I'm reverting it.